### PR TITLE
feat(postcard): postcard view and controller with tests

### PR DESCRIPTION
**Pull request by robintsunny**
what: view and controller for postcard create
why: postcard creation logic and rendering

### CODE REVIEW

**Review by vinay-pad — APPROVED** [review]
LGTM

File: test/services/controller.spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var expect = require('chai').expect;
```
Comment (jesscxu) [inline test/services/controller.spec.js:3]: nit: align `=`

File: test/services/controller.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+describe('create controller', function () {
+
+  var $controller;
+  var $q;
+  var $scope;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $controller = $injector.get('$controller');
```
Comment (jesscxu) [inline test/services/controller.spec.js:17]: nit: align `=`

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**